### ISSUE TITLE
angular-seed for ES6

### ISSUE THREAD

**Issue by jdiapps** [angular/angular-seed#260]
Has anyone re-written this for ES6? 
Thanks

**Comment by gpltaylor** [comment]
No, there is little point. I am personally waiting until Angular 2.0 hits the streets. We can then see what impact this may have. I am of course assuming a large part of Angular 2.0 will be written in ES6 using their Traceur plugin for backwards compatibility. But we can only hope...

**Comment by tolyo** [comment]
For ES6 sample, you can use [this](https://github.com/tolyo/angularjs-brunch-es6-seed).

**Comment by willyelm** [comment]
I have created a ES6 angular seed project, you'll find all the same approach of this seed(protractor, karma, etc). https://github.com/willyelm/angular-es6-seed
